go.sum

```diff
@@ -0,0 +1,2 @@
+github.com/soda-cdm/documentation v0.0.0-20230525043849-2b985763d78d h1:3xzxcT2yHfhVJWOvnVstPMU9lx3yzZ24RAojwVvHC6I=
```

Comment:
Not sure if the go.sum file should be checked in with this.

Reply:
This was auto generated

Reply:
> This was auto generated

and these can be removed after All changes can be applied as this individual does not count as a commit

Reply:
> > This was auto generated
> 
> and these can be removed after All changes can be applied as this individual does not count as a commit when doing changes from local env, it can be applied after all the changes commited

Reply:
If the go.sum file is pushed it is used when the go modules are fetched. This may cause an issue as it is looking for a specific module in the sum file.

Reply:
ok, removed it